packages/riipen-ui/src/components/Hidden.test.jsx

```diff
@@ -0,0 +1,125 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Hidden from "./Hidden.jsx";
+
+describe("<Hidden>", () => {
+  it("renders without errors", () => {
+    let error;
+    const size = "sm";
+
+    try {
+      mount(<Hidden size={size} />);
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const size = "sm";
+
+    const wrapper = mount(<Hidden size={size} />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const size = "sm";
+      const defaultProps = new Hidden({ size }).type.defaultProps;
+
+      const wrapper = mount(<Hidden size={size} />);
+
+      const component = wrapper.find("Hidden");
+      expect(component.props().direction).toEqual(defaultProps.direction);
+    });
+  });
+
+  describe("children prop", () => {
+    it("displays given children", () => {
+      const child = <h1>this is a child</h1>;
+      const size = "sm";
+
+      const wrapper = mount(<Hidden size={size}>{child}</Hidden>);
+
+      expect(
+        wrapper
+          .find("Hidden")
+          .find("div")
+          .contains(child)
+      ).toEqual(true);
+    });
+  });
+
+  describe("direction prop", () => {
+    it("applies given direction to the size-direction className on div", () => {
+      const direction = "up";
+      const size = "sm";
+      const className = new RegExp(`.+-${direction}`);
```

Comment:
I would maybe argue we don't need this in a variable and that we could just inline this value on line 69 below. The reason being we basically inline every other test outcome so might as well continue that trend.